extensions/3DTILES_implicit_tiling/schema/tile.3DTILES_implicit_tiling.templateUri.schema.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Since the template uri is really used in two places - `subtrees` and `content` - I think it makes sense to rename this to `templateUri.schema.json`.